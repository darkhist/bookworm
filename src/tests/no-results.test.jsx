import React from 'react';
import { shallow } from 'enzyme';

import NoResults from '../components/noResults/noResults';

describe('No Results', () => {
  it('should render without crashing', () => {
    const noResults = shallow(<NoResults />);
    expect(noResults).toMatchSnapshot();
  });
});
