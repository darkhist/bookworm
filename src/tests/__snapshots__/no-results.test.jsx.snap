// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Results should render without crashing 1`] = `
<styled.div>
  Sorry, nothing was found! â€‰
  <span
    aria-label="sad face emoji"
    role="img"
  >
    ðŸ˜¢
  </span>
</styled.div>
`;
